### PR TITLE
Update python version in docs

### PR DESCRIPTION
**Pull request by harupy**
## Motivation

- A follow-up for #1810
- Update python version in docs from 3.5 to 3.6.
- #1067

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 # Optuna: A hyperparameter optimization framework
 
-[![Python](https://img.shields.io/badge/python-3.5%20%7C%203.6%20%7C%203.7%20%7C%203.8-blue)](https://www.python.org)
+[![Python](https://img.shields.io/pypi/pyversions/optuna)](https://www.python.org)
```
Comment (harupy) [inline README.md:5]: This badge automatically extracts supported python versions from PyPI:

Screen Shot 2020-09-12 at 14 14 50

https://github.com/harupy/optuna/tree/update-py-version#optuna-a-hyperparameter-optimization-framework

Comment (sile) [reply README.md:5]: I didn't know the feature. It seems very convenient.

**Review by sile — APPROVED** [review]
LGTM!
Thank you for the quick follow-up.

**Review by HideakiImamura — APPROVED** [review]
Thanks for the quick action! LGTM!

**Pull request merged**